bats/1pk5col-ints.bats

```diff
@@ -129,11 +129,10 @@ if rows[2] != "9,8,7,6,5,4".split(","):
     run dolt sql -q "select * from test"
     [[ "$output" =~ "9" ]] || false
     run dolt sql -q "insert into test (c1,c3,c5) values (50,55,60)"
-    [ "$status" -eq 1 ]
-    [ "$output" = "column name 'pk' is non-nullable but attempted to set default value of null" ]
+    [[ "$output" =~ "column name 'pk' is non-nullable but attempted to set default value of null" ]]
```

Comment:
Seems like most of these changes are the sign of an issue. The error output shouldn't have changed except for syntax errors

Reply:
fixed